lldb/source/Target/Thread.cpp

```diff
@@ -617,25 +617,26 @@ void Thread::WillStop() {
   current_plan->WillStop();
 }
 
-void Thread::SetupForResume() {
+bool Thread::StepOverBreakpointIfNeeded(RunDirection direction) {
```

Comment:
That name no longer describes what this function does. It checks more things than just whether we need to step over a breakpoint, like whether this thread is suspended so we don't need to do anything, or if this is going to be a virtual step so we don't need to resume either.
SetUpForResume is a little too vague, but it's worse to say the function only does certain things when that's not in fact true.

Reply:
Renamed it to `SetupForStepOverBreakpoint()`, hope that's good.